client/verta/verta/client.py

```diff
@@ -739,8 +739,10 @@ def get_or_create_registered_model(self, name=None, desc=None, tags=None, attrs=
     def get_registered_model(self, name=None, workspace=None, id=None):
         raise NotImplementedError
 
-    def get_registered_model_version(self, id=None):
-        raise NotImplementedError
+    def get_registered_model_version(self, name=None, id=None):
+        if self._ctx.registered_model is None:
+            raise RuntimeError("Registered model is not set. Please call `client.set_registered_model()`")
+        return self._ctx.registered_model.get_version(name, id)
```

Comment:
We should actually be able to get by ID without a RegisteredModel being set, right?
I'm going to create a follow-up ticket for this, since it's not a blocker for tomorrow, and also because I think it'll involve a change to `Context.populate()`.

Reply:
Correct.

Reply:
Yep. I've created a new lower-priority ticket that should be done in a different PR.